src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
-; Please do not edit this file manually, it should only be updated through code fix application.
\ No newline at end of file
+; Please do not edit this file manually, it should only be updated through code fix application.
+### Changed Rules
+Rule ID | New Category | New Severity | Old Category | Old Severity | Notes
+--------|--------------|--------------|--------------|--------------|-------
+CA3076 | Security | Info | Security | Hidden | DoNotUseInsecureXSLTScriptExecutionAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca3076)
```

Comment:
Can we avoid this severity change please?

Reply:
Sure!